src/libraries/Microsoft.PowerFx.Connectors/Public/TabularTableValue.cs

```diff
@@ -14,22 +13,26 @@ namespace Microsoft.PowerFx.Connectors
 {
     // Created by TabularService.GetTableValue
     // Doesn't contain any ServiceProvider which is runtime only
-    public class ConnectorTableValue : TableValue, IRefreshable, IDelegatableTableValue
+    public class TabularTableValue : TableValue, IRefreshable, IDelegatableTableValue
```

Comment:
>TabularTableValue [](http://example.com/codeflow?start=17&length=17)

Can we keep old names?
"ConnectorTable" sounds better than "TabularTable"

Reply:
I prefer the separation between "Connector" prefix for action connectors and "Tabular" for what's CDP related